### ISSUE TITLE
Addition of textbox for other locations for seller

### ISSUE THREAD

**Issue by chandan2001** [sdgniser/listings#16]
### Is it necessary that only students living in hostels can sell their stuff?
If yes, then it should strictly not be the case. 
If not, then there should be an option to include a separate text box or inclusion of guest house or quarters or any other residential area for the "location" option for the welfare of students and staff members.

### Present scenario
The present version takes "somewhere else" as input for location if it is none from the listed locations. One of items used that "somewhere else" tag which is ambiguous.

**Comment by PeithonKing** [comment]
I don't think we need this at all (coz this is a lot of work to implement and no one uses it). On the other hand, if you want me to add some specific fields, I can add them right now. Adding a Text Field for that will be a lot of work with little results.
